sources/editor/Stride.Core.Assets.Editor/Services/UserDocumentationService.cs

```diff
@@ -23,57 +27,69 @@ public class UserDocumentationService
         public string GetMemberDocumentation([NotNull] IMemberDescriptor member, Type rootType)
         {
             if (member == null) throw new ArgumentNullException(nameof(member));
-            lock (lockObj)
+            string result;
+            string key;
+
+            var prefix = member is FieldDescriptor ? 'F' : 'P';
+            if (rootType != null && CacheAssemblyDocumentation(rootType.Assembly))
             {
-                string result;
-                string key;
+                // Remove generic type arguments specifications
+                key = $"{prefix}:{rootType.FullName}.{member.Name}";
 
-                var prefix = member is FieldDescriptor ? 'F' : 'P';
-                if (rootType != null)
+                lock (lockObj)
                 {
-                    if (CacheAssemblyDocumentation(rootType.Assembly))
-                    {
-
-                        // Remove generic type arguments specifications
-                        key = $"{prefix}:{rootType.FullName}.{member.Name}";
-
-                        if (cachedDocumentations.TryGetValue(key, out result))
-                            return result;
-                    }
+                    if (cachedDocumentations.TryGetValue(key, out result))
+                        return result;
                 }
+            }
 
-                if (!CacheAssemblyDocumentation(member.DeclaringType.Assembly))
-                    return null;
+            if (!CacheAssemblyDocumentation(member.DeclaringType.Assembly))
+                return null;
 
-                var declaringType = Regex.Replace(member.DeclaringType.FullName, @"\[.*\]", "");
-                // Remove generic type arguments specifications
-                key = $"{prefix}:{declaringType}.{member.Name}";
+            var declaringType = Regex.Replace(member.DeclaringType.FullName, @"\[.*\]", "").Replace('+', '.');
+
+            // Remove generic type arguments specifications
+            key = $"{prefix}:{declaringType}.{member.Name}";
 
+            lock (lockObj)
+            {
                 return cachedDocumentations.TryGetValue(key, out result) ? result : null;
-            }
+            }            
         }
 
         [CanBeNull]
         public string GetPropertyKeyDocumentation([NotNull] PropertyKey propertyKey)
         {
             if (propertyKey == null) throw new ArgumentNullException(nameof(propertyKey));
+            var ownerType = propertyKey.OwnerType;
+            if (ownerType == null)
+                return null;
+
+            if (!CacheAssemblyDocumentation(ownerType.Assembly))
+                return null;
+
+            var declaringType = ownerType.FullName;
```

Comment:
Can you also do the replace `+`, `.` here? Thanks

Reply:
I'll give it a look!